english/soar/soar.md

```diff
@@ -3,4 +3,7 @@ No, not like a bird, like a jet!
 
 With the ground moving rapidly past you.
 
-Do you:
\ No newline at end of file
+Do you:
+
+[Fly to Paris] (fly-to-paris/fly-to-paris.md)
+[Fly to London] (fly-to-london/fly-to-london.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.